hardware/arduino/avr/cores/arduino/USBCore.cpp

```diff
@@ -72,18 +65,14 @@ const u8 STRING_PRODUCT[] PROGMEM = USB_PRODUCT;
 const u8 STRING_MANUFACTURER[] PROGMEM = USB_MANUFACTURER;
 
 
-#ifdef CDC_ENABLED
```

Comment:
CDC removal was a choice (indeed the commit is atomic and can be safely reverted)